webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -172,8 +177,22 @@
                     <a href="{{ url('manage:home') }}">Admin</a>
                     {% endif %}
                     |
-                    {{ browserid_logout(text='Sign out') }}
+                    {% if request.session.get('signin_method') %}
```

Comment:
I would use `` == 'google'`` here. Just because it makes it more explicit, and more resilient to future changes.